test/e2e-common.sh

```diff
@@ -570,8 +570,11 @@ function test_mt_source() {
 
   export TEST_MT_SOURCE
 
-  echo "Run rekt tests"
-  go_test_e2e -tags=e2e -timeout=20m -test.parallel=${TEST_PARALLEL} -run "^TestKafkaSource*" ./test/rekt/... || fail_test
+  # Ginkgo is nice to have to run Ginkgo tests
+  go get github.com/onsi/ginkgo/ginkgo
+
+  echo "Run Ginkgo tests"
+  ginkgo -tags=e2e_ginkgo --randomizeAllSpecs --randomizeSuites --failOnPending --trace --race --compilers=2 ./test/e2e/source/... || fail_test
```

Comment:
I assume after this change (ginko install and this invoke), the "old" ` way to execute the "source e2e" tests will fail?

Reply:
if you mean by old way is to use `go test`, the answer is no. You can still use go test.